packages/components/src/components/Alert/Alert.tsx

```diff
@@ -76,6 +75,7 @@ export const Alert: FC<AlertProps> = ({
   testID,
 }) => {
   const colorScheme = useColorScheme()
+  const theme = useTheme()
   const fontScale = useWindowDimensions().fontScale
   const isDarkMode = colorScheme === 'dark'
```

Comment:
Not related to these changes, but did we end up duplicating logic at some point?

Searching this use--it seems like what we're doing in lines 127-134 is redundant with 231-233 and 246-248, right?

Reply:
Nice catch. Not sure when that happened. Removed duplicate logic.